### ISSUE TITLE
jsoneditor-icons.svg not loading in production

### ISSUE THREAD

**Issue by thebenedict** [eddietejeda/administrate-field-json#8]
![image](https://cloud.githubusercontent.com/assets/110948/16488959/cb13322e-3edb-11e6-9e05-65d8511bf526.png)

Are they working for anyone else? The `scripts` in package.json move jsoneditor-icons.svg to vendor/assets/images, and then `sed -i.bak 's/img/\\/assets/g'`. Since [Sprockets notoriously doesn't generate non-digested assets](https://github.com/rails/sprockets-rails/issues/49#issuecomment-20530751) in Rails 4, this simple path substitution doesn't work because a fingerprint gets added to the jsoneditor-icons.svg filename.

Am I missing something, or other ideas? If not I'll submit a fix. My first thought is to move the .svg file to /public instead of /vendor/assets/images.

**Comment by bronislav** [comment]
@thebenedict This is an issue for me as well

**Comment by bryanbraun** [comment]
Yeah, I'm seeing this too. We can get that asset to precompile by adding this line into `lib/administrate/field/json.rb`:

```ruby
Administrate::Engine.config.assets.precompile += %w( jsoneditor-icons.svg )
```

The only issue is then the CSS files are still pointing to the undigested version. Maybe we could adjust that `sed` command to update that CSS file to use the `image-url()` helper?
